src/BNO055.cpp

```diff
@@ -0,0 +1,133 @@
+#include <BNO055.hpp>
+
+IMU::BNO055::BNO055(IO::I2C& i2C) : i2c(i2C) {}
+
+/**
+ * This function is a doozie, most of the issues with it come from the fact that the BNO055 does not use standard i2c.
+ * Because of this, none of the standard EVT readReg or writeReg functions work. This function uses the write and read functions
+ * from EVT-Core to write specific bytes over i2c to correctly interact with the standard the BNO055 uses. We are not sure
+ * about what the board is using, all data was retrieved by using a Saleae device and inspecting the Arduino code.
+ */
+bool IMU::BNO055::setup() {
+    log::LOGGER.log(log::Logger::LogLevel::INFO, "Starting Initialization...\r\n");
+
+    // First write an empty i2c command to the device.
```

Comment:
Add a TODO to check if this succeeds, so we can determine if the BNO055 is connected